agent/lib/kontena/weave_adapter.rb

```diff
@@ -0,0 +1,130 @@
+require_relative 'helpers/node_helper'
+
+module Kontena
+  class WeaveAdapter
+    include Helpers::NodeHelper
+
+    WEAVE_VERSION = 'git-bd19b8625908'
```

Comment:
I would have suggested env variable as well, but I don't know is it worth of it if we don't update it so often.